library/runCmds.py

```diff
@@ -313,50 +541,78 @@ def get_x86_cpu_info(self, data, cpu_dict):
 
 
 class _GetFileRhoCmd(RhoCmd):
-    name = "file"
-    cmd_strings = []
-    filename = None
 
     def __init__(self):
+        super(_GetFileRhoCmd, self).__init__()
```

Comment:
What does this RhoCmd do? It looks like a base class but there is no documentation